templates/incl-editor-and-output.html

```diff
@@ -117,4 +117,11 @@
     </div>
     <div class="ml-3">{{ developers_mode }}</div>
   </label>
-</div>
\ No newline at end of file
+</div>
+<script>
+function ScrollOutputToBottom()
```

Comment:
Nice fix @fRedelaar, but maybe we want to place this code in app.ts rather than directly in the HTML? Or is there a reason that that is not possible?

Reply:
Nope no reason for this! I will create a new PR to place the code in app.ts. Thanks for the review, my bad!

Reply:
Moved the code at PR #1460